File: Application/ARQuizServer/entities/Question.php
<?php
/**
 * Created by PhpStorm.
 * User: Martin
 * Date: 13.2.2017
 * Time: 21:24
 */

namespace entities;


class Question
{
    public static $withOptions = 0;
    public static $writableAnswer = 1;

    private $id;
    private $name;
    private $competitionId;
    private $targetId;
    private $locationId;
    private $type;
    private $score;

    function __construct($id, $name, $competitionId, $targetId, $locationId, $type, $score)
    {
        $this->id = $id;
        $this->name = $name;
        $this->competitionId = $competitionId;
        $this->targetId = $targetId;
        $this->locationId = $locationId;
        $this->type = $type;
        $this->score = $score;
    }

    public function getId()
    {
        return $this->id;
    }

    public function getName()
    {
        return $this->name;
    }

    public function getCompetitionId()
    {
        return $this->competitionId;
    }

    public function getTargetId()
    {
        return $this->targetId;
    }

    public function getLocationId()
    {
        return $this->locationId;
    }
    public function getType()
    {
        return $this->type;
    }

    public function getScore()
    {
        return $this->score;
    }
}

File: Application/ARQuiz/app/src/main/java/com/lepko/martin/arquiz/Entities/Question.java
package com.lepko.martin.arquiz.Entities;

import java.util.List;

/**
 * Created by Martin on 6.3.2017.
 */

public class Question {

    private int id;
    private String name;
    private String targetId;
    private Location location;
    private int type;
    private int score;
    private List<Answer> answers;

    public Question(int id, String name, String targetId, Location location, int type, int score, List<Answer> answers) {
        this.id = id;
        this.name = name;
        this.targetId = targetId;
        this.location = location;
        this.type = type;
        this.score = score;
        this.answers = answers;
    }

    public String getName() {
        return name;
    }

    public int getId() {
        return id;
    }

    public String getTargetId() {
        return targetId;
    }

    public Location getLocation() {
        return location;
    }

    public int getType() {
        return type;
    }

    public List<Answer> getAnswers() {
        return answers;
    }

    public int getScore() {
        return score;
    }
}
